### DISCUSSION TITLE
必要な機能のアイデア出し

### DISCUSSION THREAD

**Discussion by t-sakurai816** [t-sakurai816/inYourFreeTime#2] [Ideas]
趣味の共有ができるアプリに必要な機能のアイデア出しを行います

**Comment by t-sakurai816** [comment]
# 超基本的な機能

## 閲覧

絶対必要

## 投稿

絶対必要

## 編集

あればいいくらい？優先度低い

## 削除

あればいいくらい？優先度低い

**Reply by yCroma** [reply]
プロフィールとか設定できるようにするん？
一回投稿したら、そこで完結みたいにするん？
二回目以降の投稿とかについては、どんな方針でいこうと思っとるん？

最小限のリリースのみを考えれば、投稿しっぱなし、一回投稿したらそれで完了みたいな感じ？

あとはひとつアイデアとしては、Idは早い者勝ち、投稿時に各自入力、ユーザー登録が完了したら、他人は自分のid で投稿できなくなるとか？

**Reply by t-sakurai816** [reply]
とにかく認証はやりたくないから、

投稿時にプロフィールを設定できる

Ex)

```
name（必須）: 匿名太郎
趣味（必須）：キャンプ
内容（必須）：車で山へ行き、焚き火でご飯を食べるのが好きです
年齢（任意）：27歳
性別（任意）：男
```

みたいにするのはどうだろう？
何回目の投稿だろうと、そこは管理しない。
idとかもいらない。


その投稿に対するいいねくらいはつけてもいいかも

**Reply by yCroma** [reply]
OK!
じゃあ、大事なことはデザインとセキュリティか
うーん、がんばろ；；
